indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -34,68 +34,108 @@
                         {% for abstract in abstracts %}
                             <tr class="i-table abstract-row"
                                 data-friendly-id="{{ abstract.friendly_id }}"
-                                data-title="{{ abstract.as_legacy.getTitle() }}">
+                                data-title="{{ abstract.title }}">
                                 <td class="i-table id-column">
-                                    <span class="vertical-aligner">
-                                        <input type="checkbox" class="select-row" name="abstract_id"
-                                               value="{{ abstract.id }}">
-                                    </span>
+                                    <input type="checkbox" class="select-row" name="abstract_id"
+                                           value="{{ abstract.id }}">
```

Comment:
ah.. looked shorter. we don't have a strict limit as we do in python but we usually try to keep around <=120 too. for 124 i would decide case by case whether wrapping makes it more or less readable. in this case i don't have a strong opinion so i'd probably just keep it as it is now